src/main/js/main_html5.js

```diff
@@ -455,17 +455,24 @@ require("../../../html5-common/js/utils/environment.js");
       // In this case we allow setting the initial time back to 0 as a workaround for this
       var queuedSeekRequired = OO.isSafari && videoEnded && time === 0;
 
-      initialTime.value = time;
-      initialTime.reached = false;
-
-      // [PBW-3866] Some Android devices (mostly Nexus) cannot be seeked too early or the seeked event is
-      // never raised, even if the seekable property returns an endtime greater than the seek time.
-      // To avoid this, save seeking information for use later.
-      // [PBW-5539] Same issue with desktop Safari when setting initialTime after video ends
-      if (OO.isAndroid || (queuedSeekRequired && !OO.isIos)) {
-        queueSeek(initialTime.value);
-      } else {
-        this.seek(initialTime.value);
+      // [PBW-7473] On replay, there is a bug on IE11 where the player won't
+      // send out some messages from the video element causing the UI to keep
+      // the replay screen up.
+      var doNotSeek = OO.isIE11Plus && hasPlayed && time === 0;
```

Comment:
Can we do the queue Seeking that we do in the comment below (starting with "[PBW-3866]"? It sounds like these could be the same issue.

Reply:
Will do, looks like that's the better solution. yay code reviews!